packages/manager/apps/telecom/src/app/telecom/pack/xdsl/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+import component from './pack-xdsl.component';
+import routing from './pack-xdsl.routing';
+import templates from './pack-xdsl.templates';
+
+const moduleName = 'ovhManagerTelecomPackXdsl';
+
+angular
+  .module(moduleName, [])
+  .component('xdsl', component)
```

Comment:
Why not using `packXdsl` as a component name?

I find it more suitable with the actual structure. Do you agree?

Reply:
you're right.